packages/IMAPClient-UI.package/ICFolderDialog.class/instance/selectedEndpoint..st

```diff
@@ -4,4 +4,6 @@ selectedEndpoint: anICEndpoint
 	(anICEndpoint = self selectedEndpoint)
 		ifFalse: [
 			selectedEndpoint := anICEndpoint.
-			self switchToNewEndpoint]
\ No newline at end of file
+			self switchToNewEndpoint].
+		
+	self changed: #selectedEndpoint
```

Comment:
I changed the changes to the setter methods if possible since it can be reused better in the future and a setter change should result in an update of the corresponding UI.

Reply:
Yeah this makes sense to me. thought `#switchToNewEndpoint` would do something different.